data/ecosystem/vendors.yaml

```diff
@@ -1,3 +1,9 @@
+- name: Apica
+  nativeOTLP: true
+  url: https://docs.apica.io/technologies/ascent-with-opentelemetry
+  contact:
```

Comment:
please add a point of contact here, can be an email or a github handle, we might use this in the future to let you know if anything changes